kf_lib_data_ingest/common/pandas_utils.py

```diff
@@ -244,8 +252,19 @@ def resolve_duplicates(df, suffixes):
         return df
 
     merged = pandas.merge(left, right, **kwargs)
+    reduced = resolve_duplicates(merged, kwargs.pop('suffixes', ('_x', '_y')))
+
+    msg = (
+        f'*** {kwargs.get("how", "inner").title()} merge {left_name} with '
```

Comment:
I guess "inner" is from `inspect.signature(pandas.merge).parameters['how'].default`?

Reply:
👍